newrelic/hooks/mlmodel_sklearn.py

```diff
@@ -80,12 +80,52 @@ def _nr_wrapper_method(wrapped, instance, args, kwargs):
         if method in ("predict", "fit_predict"):
             training_step = getattr(instance, "_nr_wrapped_training_step", "Unknown")
             wrap_predict(transaction, _class, wrapped, instance, args, kwargs)
+            create_label_event(transaction, _class, instance, return_val)
             return PredictReturnTypeProxy(return_val, model_name=_class, training_step=training_step)
         return return_val
 
     wrap_function_wrapper(module, "%s.%s" % (_class, method), _nr_wrapper_method)
 
 
+def create_label_event(transaction, _class, instance, return_val):
+    inference_id = uuid.uuid4()
+    model_name = getattr(instance, "_nr_wrapped_name", _class)
+    model_version = getattr(instance, "_nr_wrapped_version", "0.0.0")
+    label_names = getattr(instance, "_nr_wrapped_label_names", None)
+    settings = transaction.settings if transaction.settings is not None else global_settings()
+
+    if return_val is not None:
```

Comment:
You can just shorten this to:
```suggestion
    if return_val:
```

Reply:
Good call out! This was how I initially had this implemented but since `return_val` is an `ndarray`, numpy throws the following error: `The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`. To get around this, I had to explicitly call out the `None` case.

Reply:
Oh right! I've run into this one before-I totally forgot about it. Ndarrays and their special handling... :/